src/wily/__main__.py

```diff
@@ -415,14 +415,24 @@ def diff(ctx, files, metrics, all, detail, revision, wrap):
     default=False,
     help=_("Aggregate if path is directory"),
 )
+@click.option(
+    "--plotlyjs",
+    default="True",
```

Comment:
Please use a bool parameter type (see https://click.palletsprojects.com/en/8.1.x/parameters/#parameter-types) then you can remove the code below that converts the string back to a bool

Reply:
I think 2 additional bool flags

`--shared-js` for `directory` and `--cdn-js` for the CDN option. If either of those are set, they will override the default option (of true)